content/development/handbook/front-end/build/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Build
+description: Build documentation for altinn.studio frontend
+tags: ["development", "frontend", "build"]
+weight: 100
+---
+
+### Building of react apps in altinn.studio
+
+The building of react apps are done in dockerfiles which then again are used in the build of altinn-core docker-image.  
+Apps are compiled and copied.
+
+### Adding new packages to react-applications
+
+Adding of a new package in an app is done with `npm` in the app where it's needed. If multiple apps have the same package as a dependency, lerna will hoist the package, so it will only be installed once, and used by the depending apps.
```

Comment:
isnt it best practice to use lerna to add new packages to the react-application?

Reply:
Like, `lerna exec npm install [package-name]`? 
That would install that package in every react application we have. Which is think is not wanted.

Reply:
No 'lerna add [package] --scope='project' --exact' 
https://github.com/lerna/lerna/tree/master/commands/add#readme

Reply:
Added to docs